src/app/public/params.ts

```diff
@@ -132,13 +134,13 @@ export class SkyAppRuntimeConfigParams {
    * @param urlDecode A flag indicating whether the value should be URL-decoded.
    * Specify true when you anticipate the value of the parameter coming from the page's URL.
    */
-  public get(key: string, urlDecode?: boolean): string {
+  public get(key: string, urlDecode: boolean = false): string {
     if (this.has(key)) {
       let val = this.params[key];
 
       // This should be changed to always decode encoded params in skyux-builder 2.0.
-      if (urlDecode && this.encodedParams.indexOf(key) >= 0) {
-        val = decodeURIComponent(val);
+      if (!urlDecode && this.encodedParams.indexOf(key) >= 0) {
```

Comment:
Why change the logic here to a `!` & `encodeURIComponent`?

Reply:
The newer URL params utility does not encode the params by default, but we need to avoid breaking changes, which is why I flipped the logic. However, to your point, it doesn’t make much sense to reverse the logic and keep the variable names the same (since it doesn’t make sense semantically anymore). I moved the backward-compatible logic to another location and added a comment for posterity.